aiowebostv/exceptions.py

```diff
@@ -1,17 +1,17 @@
 """Exceptions for aiowebostv."""
 
 
-class PyLGTVPairException(Exception):
+class WebOsTvPairException(Exception):
     """Exception raised to represent TV pairing errors."""
 
 
-class PyLGTVCmdException(Exception):
+class WebOsTvCmdException(Exception):
     """Exception raised to represent TV command exceptions."""
 
 
-class PyLGTVCmdError(PyLGTVCmdException):
+class WebOsTvCmdError(WebOsTvCmdException):
```

Comment:
Is there a difference between this error and WebOsTvCmdException?

Reply:
Yes, I had a thought to remove the 2nd one, but it does has a small difference, this is the relevant code:
https://github.com/home-assistant-libs/aiowebostv/blob/3d13071d2b3231498f10af7b6f8147ff8648df2d/aiowebostv/webos_client.py#L618-L622


`WebOsTvCmdException` raised to indicate missing or invalid responses (such as transport errors), while the `WebOsTvCmdError` indicates that the device did get the request but responded with something indicates that the request was not accepted. 

I did receive a `WebOsTvCmdError` during debugging since I had a problem in the request payload. Currently we only catch `WebOsTvCmdException` and in some cases silently drop them (for example TV is off - no connection).

I think it is a good practice to raise the `WebOsTvCmdError` error if in the future something will break it will be logged immediately since we don't catch this error.

Reply:
Ok. Can we rename one or both of these exceptions to make it more clear what they represent? Maybe WebOsTbCmdInvalid?

Reply:
Updated PR description with renamed exceptions